File: Competition.js
// making Competition class that have function addTeam, addGame.
class Team {
	constructor (newRank = 0, newName = 'unnamed', newVenue = 'place', newCity = 'city') {
		this.rank = newRank
		this.name = newName
		this.venue = newVenue
		this.city = newCity
	}

	toString () {
    let result = `${this.name} is at rank @ ( ${this.rank} ) with venue $${this.venue.toFixed(2)} from city ${this.city}`
    return result
	}
}

class Game {
	constructor (newWeek = 0, newHomeTeamRank = 0, newAwayTeamRank = 0, newDateTime = 'untimed'){
		this.week = newWeek
		this.homeTeamRank = newHomeTeamRank
		this.awayTeamRank = newAwayTeamRank
		this.dateTime = newDateTime
	}
	toString () {
    let result = `@ ${this.week} there is a game between ( ${this.homeTeamRank} ) versus $${this.awayTeamRank} on ${this.dateTime}`
    return result
	}
}


class Competition {
	constructor () {
		this.allTeams = []
		this.allGames = []
	}

	addTeam(newRank, newName, newVenue, newCity){
		let newTeam = new Team(newRank, newname, newVenue, newCity)
		this.allTeams.push(newTeam)
	}

	addGame(newWeek, newHomeTeamRank, newAwayTeamRank, newDateTime){
		let  newGame = new Game(newWeek, newHomeTeamRank, newAwayTeamRank, newDateTime)
		this.allGames.push(newGame)
	}
}

File: competition.py
class Team:
    def __init__(self, newRank, newName, newVenue, newCity):
        self.rank = newRank
        self.name = newName
        self.venue = newVenue
        self.city = newCity

    def __str__(self):
        result  = "testing"
        return result

class Game:
    def __init__(self, newWeek, newHomeTeamRank, newAwayTeamRank, newDateTime):
        self.week = newWeek
        self.homeTeamRank = newHomeTeamRank
        self.awayTeamRank = newAwayTeamRank
        self.dateTime = newDateTime

    def __str__(self):
        result = "testing"
        return result


class Competition:
    def __init__(self, new_title, new_year):
        self.title = new_title
        self.year = new_year
        self.allTeams = []
        self.allGames = []

    def addTeam(self, newRank, newName, newVenue, newCity):
        newTeam = Team(newRank, newName, newVenue, newCity)
        self.allTeams.append(newTeam)

    def addGame(self, newWeek, newHomeTeamRank, newAwayTeamRank, newDateTime):
        newGame = Game(newWeek, newHomeTeamRank, newAwayTeamRank, newDateTime)
        self.allGames.append(newGame)

    def __str__(self):
        result = "testing"
        return result
